assemble-and-restart

```diff
@@ -2,6 +2,18 @@
 set -x
 set -eo pipefail
 
+export ODO_SUPERVISORD_DIR=/var/lib/supervisord/
+export ODO_IMAGE_MAPPINGS_FILE=${ODO_SUPERVISORD_DIR}/language-scripts/image-mappings.json
+
+IMAGE_LANG=`${ODO_SUPERVISORD_DIR}/bin/getlanguage`
+
+if [ ! -z "${IMAGE_LANG}" ]; then
+    ${ODO_SUPERVISORD_DIR}/language-scripts/${IMAGE_LANG}/dev-assemble
```

Comment:
I am not too familiar with bash, but what will happen if IMAGE_LANG is not "" and also the script is not found?

Reply:
if `IMAGE_LANG` is not "" the `[ ! -z "${IMAGE_LANG}" ];` evaluates as `true`

if the script is not found the command will fail

Reply:
> if the script is not found the command will fail

Will it crash?

Reply:
yes, it will fail, as it is an error state

Reply:
> yes, it will fail, as it is an error state

Will it be handled?

Reply:
It will be handled by erroring out :-) This is a state that we can't safely recover from.

Reply:
And will the normal scripts be run if this one failed?

Reply:
No and we don't want that. This would lead to unpredictable errors, that we can't catch or hide other important errors.